GA-worksheet.md

```diff
@@ -6,18 +6,18 @@
 
 
 ##Project #2 Part #1: Header HTML
-1. What is the purpose of the header tag?
+1. What is the purpose of the header tag? To put headers inside of. It's for headers.
 
-2. What is a Navigation Bar?
+2. What is a Navigation Bar? A navigation bar is like a "portal" to a list of links, to help users of the website access           specific information easier.
 
-3. What tag do we use to make a navigation bar? <      >
+3. What tag do we use to make a navigation bar? you use a { } or use <ul> and <li>.
```

Comment:
On the HTML side, we use <ul> and <li>. On the CSS side, we select our HTML element and add in our CSS properties and values using curly braces { }.